### PR TITLE
[bugfix] bugfix different version data and select/measure layer problem

### PR DESCRIPTION
**Pull request by Palad1NPT**
[bugfix] bugfix different version data and select/measure layer problem

### CODE REVIEW

File: Fog.js [hunk 1]
```diff
@@ -868,7 +868,11 @@ function setup_draw_buttons() {
 
 
 		var target = $("#fog_overlay");
-		target.css("z-index", "50");
+		if (!e.currentTarget.id || (e.currentTarget.id !== "measure-button" && e.currentTarget.id !== "select-button")) {
+			target.css("z-index", "50");
+		} else {
+			target.css("z-index", "20");
+		}
```
Comment (MrKuzio) [inline Fog.js:875]: What are you trying to do here? 🤔 
Can you run me through the logic, please?

Comment (Palad1NPT) [reply Fog.js:875]: yes, to be able to draw over tokens, I set z-indexes to bring the drawing layer on top of the tokens layer.
for the measure and select this doesn't apply as these tools interact directly with the tokens.

Comment (cyruzzo) [reply Fog.js:875]: I think it's better that we definitely change the behaviour for "measure" to stop it interacting with tokens.
So this "exception" for change of z-index should only be for "select".

I think in the future we should create some ui to display token movement directly inside the draggable options of the token.. so everytime you move a token you alway get that measurament without having to go to the measure tool.

Comment (Palad1NPT) [reply Fog.js:875]: cool, going to make it only for the select then!

Comment (Palad1NPT) [reply Fog.js]: done

**Pull request merged**